tests/unit/amo/sagas/test_home.js

```diff
@@ -67,9 +67,9 @@ describe(__filename, () => {
       const state = sagaTester.getState();
 
       const firstCollectionSlug = 'collection-slug';
-      const firstCollectionUser = 'user-id-or-name';
+      const firstCollectionUser = 123;
```

Comment:
For clarity, do you think these should be called `firstCollectionUserId` and `secondCollectionUserId`?

Reply:
Yep, thanks!